### PR TITLE
Adding custom properties to `forms.css`

### PR DESCRIPTION
**Pull request by danfarrow**
Working on adding custom properties to `forms.css`. I’ll move my new properties into `custom-properties.css` once I’ve finished adding them

Trac ticket: [#49930 - Replace wp-admin color schemes with CSS custom properties](https://core.trac.wordpress.org/ticket/49930#no1)

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

**Comment by danfarrow** [comment]
All hex colour values now changed to custom properties, and all custom properties moved to the bottom of  `custom-properties.css` 

I used the prefix `--wp-admin--form-fields--` for most except where I found related custom-props that had already been defined in `custom-properties.css`. For example `--wp-admin--input--color` and  `--wp-admin--input--color--placeholder` were already defined, so I added `--wp-admin--input--color--disabled` and `--wp-admin--input--color--invalid`

My strategy for deciding to either (A) use an existing custom-prop, (B) add a new custom prop aliasing an existing custom-prop or (C) use a new custom prop with the same value as an existing custom prop was a little random, so there are a couple of instances of (C) in the props I’ve added.

I’ve ignored `rgba` values for now!

File: src/wp-admin/css/forms.css [hunk 1]
```diff
@@ -101,8 +101,8 @@ input[type="checkbox"]:focus,
 input[type="radio"]:focus,
 select:focus,
 textarea:focus {
-	border-color: #2271b1;
-	box-shadow: 0 0 0 1px #2271b1;
+	border-color: var(--wp-admin--form-fields--border--focus);
```
Comment (0aveRyan) [inline src/wp-admin/css/forms.css:104]: @danfarrow is there a reason for separating `--wp-admin--form-fields--border--focus` and `--wp-admin-theme-primary` in this selector declaration?

Comment (danfarrow) [reply src/wp-admin/css/forms.css:104]: No there is no special reason, I think at that point I hadn’t decided whether to make aliases or not. I will try & remedy it!

**Comment by danfarrow** [comment]
Closing this due to git confusion on my part. I’ve opened a new PR here https://github.com/ryelle/wordpress-develop/pull/28

**Pull request closed without merging**